expressjs-backend/models/task-services.js

```diff
@@ -55,6 +55,15 @@ async function findTaskById(id) {
   }
 }
 
+async function deleteCompeteByUser(user) {
```

Comment:
Should this say deleteCompleteByUser? Looks like it works, not a big deal

Reply:
yes it should. thanks for catching that